cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/service/checker/GcpDatabaseResourceChecker.java

```diff
@@ -36,13 +49,26 @@ protected void checkSqlOperationError(com.google.api.services.sqladmin.model.Ope
             String msg = null;
             StringBuilder error = new StringBuilder();
             if (execute.getError().getErrors() != null) {
+                LOGGER.info("Gcp database operation response contains errors.");
                 for (OperationError operationError : execute.getError().getErrors()) {
                     error.append(String.format("code: %s -> message: %s %s",
                             operationError.getCode(), operationError.getMessage(), System.lineSeparator()));
                 }
                 msg = error.toString();
+                LOGGER.warn("Gcp database operation failed and return with the following errors: '{}'", msg);
             }
             throw new CloudConnectorException(msg);
         }
     }
+
+    @VisibleForTesting
+    SQLAdmin.Operations.Get getSqlAdminOperations(SQLAdmin sqlAdmin, String operationId, String projectId) throws IOException {
+        return GcpStackUtil.sqlAdminOperations(sqlAdmin, projectId, operationId);
```

Comment:
maybe better to write the test for the static method

Reply:
I like this approach 😄

Reply:
I mean, write the test for this class: `GcpStackUtil`. You don't mock the static method neither here nor there

Reply:
and you don't need to set the modifier to package protected if you move the test to `GcpStackUtilTest`